Sources/Perform/UIViewController.swift

```diff
@@ -28,3 +28,12 @@ extension UIViewController {
     }
   }
 }
+
+#if swift(>=4.2)
```

Comment:
I think this could be `#if !swift(>=4.2)` maybe, to remove the need for `#else`?